sdk/docs/manually-written/overview/overview/unifi.rst

```diff
@@ -1,59 +1,33 @@
-Unifi
-#####
+The enterprise suite for Canton
+###############################
 
 .. wip::
    * CREATE A VIDEO / ANIMATED RECORDING
-     
-   * REVIEW the open source vs enterprise split.
-     right now its very canton focused.
-     zero about utilties
 
    * Review the downloading section.
      Add CN artifacts if there are any offerings on that,
      refer to https://docs.sync.global/ for OSS docs
 
 
 ..
-  What is Unifi?
-  **************
+Digital Asset’s enterprise products
+***********************************
```

Comment:
Looks like the Sphinx build is complaining because of indentation changes on these lines:

```suggestion
  Digital Asset’s enterprise products
  ***********************************
```